src/polyswarm/__main__.py

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env python3
-from .base import polyswarm
+from __future__ import absolute_import
+import os
+import sys
+from polyswarm.client.polyswarm import polyswarm
+
+
+def main():
+    # fixing the name of the entrypoint when the module is executed as
+    # python -m s3conf
```

Comment:
what is s3conf?

Reply:
Ops, sorry, this other codebase of mine which I used as template.